llvm/tools/llvm-reduce/deltas/ReduceOperands.cpp

```diff
@@ -168,3 +166,18 @@ void llvm::reduceOperandsNaNDeltaPass(Oracle &O, ReducerWorkItem &WorkItem) {
   };
   extractOperandsFromModule(O, WorkItem, ReduceValue);
 }
+
+void llvm::reduceOperandsPoisonDeltaPass(Oracle &O, ReducerWorkItem &WorkItem) {
```

Comment:
can you add a comment that we're only doing this for non-constant TargetExtTypes?